### ISSUE TITLE
[BUG] Installed extension still showing blue "Install" button on the detail page

### ISSUE THREAD

**Issue by nguyenhongphat0** [opensumi/core#1394]
After installing an extension from the Extension Market, the extension is installed, but the button is still showing `Install`, confusing the user that the extension hadn't installed or failed somehow 🤔 

**复现路径（To Reproduce）**

1. Go to 'Extension Market'
2. Search any extension
3. Click on 'Install', and wait for the extension to be installed
4. The button is still Install, and there is no way to uninstall the extension5. 

**预期表现（Expected behavior）**

Both button above should show "Installed", and there should be a dropdown to "Remove" the extension (like VS Code).

**问题截图（Screenshots）**

<img width="1552" alt="Screen Shot 2022-07-20 at 17 26 59" src="https://user-images.githubusercontent.com/29925961/179960761-5ff9298f-d482-4c34-8f15-d5d7d087afbc.png">

**环境信息（Environment）**

- OS: [macOS 12.4 Apple M1]
- OpenSumi Version: [2.6.10]
- Full dependencies versions: 
```json
{
    "@opensumi/di": "^1.1.0",
    "@opensumi/ide-addons": "2.16.10",
    "@opensumi/ide-comments": "2.16.10",
    "@opensumi/ide-core-browser": "2.16.10",
    "@opensumi/ide-core-common": "2.16.10",
    "@opensumi/ide-core-electron-main": "2.16.10",
    "@opensumi/ide-core-node": "2.16.10",
    "@opensumi/ide-debug": "2.16.10",
    "@opensumi/ide-decoration": "2.16.10",
    "@opensumi/ide-editor": "2.16.10",
    "@opensumi/ide-electron-basic": "2.16.10",
    "@opensumi/ide-explorer": "2.16.10",
    "@opensumi/ide-extension": "2.16.10",
    "@opensumi/ide-extension-manager": "2.16.10",
    "@opensumi/ide-extension-storage": "2.16.10",
    "@opensumi/ide-file-scheme": "2.16.10",
    "@opensumi/ide-file-search": "2.16.10",
    "@opensumi/ide-file-service": "2.16.10",
    "@opensumi/ide-file-tree-next": "2.16.10",
    "@opensumi/ide-i18n": "2.16.10",
    "@opensumi/ide-keymaps": "2.16.10",
    "@opensumi/ide-logs": "2.16.10",
    "@opensumi/ide-main-layout": "2.16.10",
    "@opensumi/ide-markdown": "2.16.10",
    "@opensumi/ide-markers": "2.16.10",
    "@opensumi/ide-menu-bar": "2.16.10",
    "@opensumi/ide-monaco": "2.16.10",
    "@opensumi/ide-monaco-enhance": "2.16.10",
    "@opensumi/ide-opened-editor": "2.16.10",
    "@opensumi/ide-outline": "2.16.10",
    "@opensumi/ide-output": "2.16.10",
    "@opensumi/ide-overlay": "2.16.10",
    "@opensumi/ide-preferences": "2.16.10",
    "@opensumi/ide-process": "2.16.10",
    "@opensumi/ide-quick-open": "2.16.10",
    "@opensumi/ide-scm": "2.16.10",
    "@opensumi/ide-search": "2.16.10",
    "@opensumi/ide-static-resource": "2.16.10",
    "@opensumi/ide-status-bar": "2.16.10",
    "@opensumi/ide-storage": "2.16.10",
    "@opensumi/ide-task": "2.16.10",
    "@opensumi/ide-terminal-next": "2.16.10",
    "@opensumi/ide-testing": "2.16.10",
    "@opensumi/ide-theme": "2.16.10",
    "@opensumi/ide-toolbar": "2.16.10",
    "@opensumi/ide-userstorage": "2.16.10",
    "@opensumi/ide-utils": "2.16.10",
    "@opensumi/ide-variable": "2.16.10",
    "@opensumi/ide-webview": "2.16.10",
    "@opensumi/ide-workspace": "2.16.10",
    "@opensumi/ide-workspace-edit": "2.16.10",
    "@typescript-eslint/eslint-plugin": "^5.15.0",
    "@typescript-eslint/parser": "^5.15.0",
    "ajv": "^8.11.0",
    "await-event": "^2.1.0",
    "buffer": "^6.0.3",
    "copy-webpack-plugin": "^11.0.0",
    "cross-env": "^7.0.3",
    "css-loader": "^6.7.1",
    "electron": "18.3.4",
    "electron-builder": "^23.0.3",
    "electron-devtools-installer": "^3.2.0",
    "electron-rebuild": "^3.2.7",
    "eslint": "^8.11.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-react": "^7.23.2",
    "fs-extra": "^8.1.0",
    "html-webpack-plugin": "^5.5.0",
    "husky": "^7.0.0",
    "less": "^4.1.2",
    "less-loader": "^11.0.0",
    "lint-staged": "^12.3.5",
    "mini-css-extract-plugin": "^2.6.0",
    "mobx": "^5.9.4",
    "mobx-react-lite": "^1.3.1",
    "node-fetch": "^2.6.1",
    "npm-run": "^5.0.1",
    "npm-run-all": "^4.1.5",
    "null-loader": "^4.0.1",
    "prettier": "^2.5.1",
    "process": "^0.11.10",
    "react-dom": "^16.8.6",
    "request": "^2.88.2",
    "rimraf": "^3.0.0",
    "shell-env": "^4.0.1",
    "style-loader": "^0.23.1",
    "style-resources-loader": "^1.2.1",
    "tailwindcss": "^3.1.5",
    "ts-loader": "^9.3.0",
    "ts-node": "8.0.2",
    "tsconfig-paths": "^3.8.0",
    "tsconfig-paths-webpack-plugin": "^3.2.0",
    "typescript": "^4.6.2",
    "webpack": "^5.72.1",
    "webpack-cli": "^4.9.1",
    "webpack-node-externals": "^3.0.0",
    "zmp-cli": "file:./../zmp-cli"
  }
```
**补充信息（Additional context）**

**Comment by erha19** [comment]
yes, It should show the same status on the button, and it also need a `uninstall` button, we will do it late, maybe next week.

If someone wants to fix it, feel free to ask more for the detail.

**Comment by erha19** [comment]
@nguyenhongphat0 hi, I have fixed this on [#1855](https://github.com/opensumi/core/pull/1855), you can use 2.21.* version to fix it.
